dsl/properties/scripts/DockerClient.groovy

```diff
@@ -148,18 +148,18 @@ public class DockerClient extends BaseClient {
             createOrUpdateService(clusterEndpoint, serviceDetails)
         }
         
-        /*
+        def uri = new URI(clusterEndpoint)
+        def serviceEndpoint = "http://${uri.host}"
```

Comment:
Do we also support the cluster end point with the protocol? In that case we would need to account for it when building the serviceEndPoint url. We should also support https.

Reply:
@urvashisingh , You are right. We do not support cluster end point with protocol. Mentioning "http://" would be bad idea. Removing it from here and keeping it same like other plugins.